Bottom: 9d323ba188206d78961ab015d9c546b2d79485ab
Top:    67887e18f95733eeb0f54c46931e83d9c6acf893
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-05-06 17:01:34 +0100

Refresh of add-multi-clearance-drc-whilst

---

diff --git a/src/file.c b/src/file.c
index 657efeb..6861569 100644
--- a/src/file.c
+++ b/src/file.c
@@ -664,6 +664,11 @@ WritePCBNetlistData (FILE * FP)
 	  PrintQuotedString(FP, &menu->Name[2]);
 	  fprintf (FP, " ");
 	  PrintQuotedString(FP, (char *)UNKNOWN (menu->Style));
+	  if (menu->Netclass != NULL)
+	    {
+	      fprintf (FP, " ");
+	      PrintQuotedString(FP, menu->Netclass);
+	    }
 	  fprintf (FP, ")\n\t(\n");
 	  for (p = 0; p < menu->EntryN; p++)
 	    {
